### PR TITLE
Avoid ce fails on windows.

### PR DESCRIPTION
**Pull request by wzzju**

### CODE REVIEW

**Review by wopeizl — DISMISSED** [review]

File: python/paddle/fluid/contrib/slim/tests/test_quantization_pass.py [hunk 1]
```diff
@@ -138,29 +138,29 @@ def linear_fc_quant(self, quant_type):
             place=place,
             activation_quantize_type=quant_type)
         transform_pass.apply(graph)
-        marked_nodes = set()
-        for op in graph.all_op_nodes():
-            if op.name().find('quantize') > -1:
-                marked_nodes.add(op)
-        graph.draw('.', 'quantize_fc_' + quant_type, marked_nodes)
+        if not enable_ce:
```
Comment (JiabinYang) [inline python/paddle/fluid/contrib/slim/tests/test_quantization_pass.py]: will this enable in windows CI, it looks like only disabled in ce?

**Review by JiabinYang — DISMISSED** [review]
LGTM

File: python/paddle/fluid/contrib/slim/tests/test_quantization_pass.py (hunk 1 above)
Comment (wzzju) [reply python/paddle/fluid/contrib/slim/tests/test_quantization_pass.py]: Yes, maybe the name is confusing. I will change it.

**Review by JiabinYang — APPROVED** [review]
Excellent

**Pull request merged**